ws-client/ws.q

```diff
@@ -21,4 +21,12 @@ open0:{[x;y;v]
 
 open:{neg first open0[x;y;.ws.VERBOSE]}                                 //return neg handle for messaging
 
+.ws.close:{[h]
+  if[all(h in key .ws.w;h in key .z.W);hclose h];                       //close handle if h is found both in .ws.w and .z.W (all opened handles)
```

Comment:
I think we should use `abs h` as within the lib we typically deal with neg handle but I think `hclose` requires positive

```suggestion
  h:abs h;
  if[all(h in key .ws.w;h in key .z.W);hclose h];                       //close handle if h is found both in .ws.w and .z.W (all opened handles)
```